test/conformance/ingress/run.go

```diff
@@ -47,4 +54,12 @@ func RunConformance(t *testing.T) {
 
 	t.Run("websocket", TestWebsocket)
 	t.Run("websocket/split", TestWebsocketSplit)
+
+	if test.ServingFlags.EnableBetaFeatures {
+		// Add your conformance test for beta features
+	}
+
+	if test.ServingFlags.EnableAlphaFeatures {
+		// Add your conformance test for alpha features
+	}
```

Comment:
A few things:

1. I wonder if for "conformance" it makes more sense to orient this in terms of Must/Should/May instead of alpha/beta (these may be correlated, but we haven't established that).  I now realize this is ingress conformance where our specification is...  lacking.  (cc @tcnghia)

2. I'm wondering whether we really want a shared entrypoint, since often for these features (esp. alpha) a flag must be flipped for the feature to be accessible, and we may not want the fuller tests to run with that flag enabled.

Reply:
For 1) I think when we introduce new fields in a spec regardless of their requirement level we'd want inclusion to be gradual to give downstream implementors time to adopt.

2) That doesn't happen all the time

> and we may not want the fuller tests to run with that flag enabled.